src/lib/Form/Data/Search/TrashSearchData.php

```diff
@@ -0,0 +1,221 @@
+        $this->contentType = $contentType;
+        $this->trashed = $trashed;
+        $this->trashedInterval = $trashedInterval;
+        $this->creator = $creator;
+        $this->sort = $sort;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getTrashed(): ?string
+    {
+        return $this->trashed;
+    }
+
+    /**
+     * @param string|null $trashed
+     */
+    public function setTrashed(?string $trashed): void
+    {
+        $this->trashed = $trashed;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getLimit(): ?int
+    {
+        return $this->limit;
+    }
+
+    /**
+     * @param int|null $limit
+     */
+    public function setLimit(?int $limit): void
+    {
+        $this->limit = $limit;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getPage(): ?int
+    {
+        return $this->page;
+    }
+
+    /**
+     * @param int|null $page
+     */
+    public function setPage(?int $page): void
+    {
+        $this->page = $page;
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\Content\Section|null
+     */
+    public function getSection(): ?Section
+    {
+        return $this->section;
+    }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Section|null $section
+     */
+    public function setSection(?Section $section): void
+    {
+        $this->section = $section;
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\ContentType\ContentType|null
+     */
+    public function getContentType(): ?ContentType
+    {
+        return $this->contentType;
+    }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType|null $contentType
+     */
+    public function setContentType(?ContentType $contentType): void
+    {
+        $this->contentType = $contentType;
+    }
+
+    /**
+     * @return array|null
+     */
+    public function getTrashedInterval(): ?array
+    {
+        return $this->trashedInterval;
+    }
+
+    /**
+     * @param array|null $trashedInterval
+     */
+    public function setTrashedInterval(?array $trashedInterval): void
+    {
+        $this->trashedInterval = $trashedInterval;
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\User\User|null
+     */
+    public function getCreator(): ?User
+    {
+        return $this->creator;
+    }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\User\User|null $creator
+     */
+    public function setCreator(?User $creator): void
+    {
+        $this->creator = $creator;
+    }
+
+    /**
+     * @return array|null
+     */
+    public function getSort(): ?array
+    {
+        return $this->sort;
+    }
+
+    /**
+     * @param array|null $sort
+     */
+    public function setSort(?array $sort): void
+    {
+        $this->sort = $sort;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isFiltered(): bool
+    {
+        $contentTypes = $this->getContentType();
```

Comment:
```suggestion
        $contentType = $this->getContentType();
```
this is one CT or null